budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/service/notification/LimitReachedNotification.java

```diff
@@ -0,0 +1,28 @@
+package org.wickedsource.budgeteer.service.notification;
+
+public class LimitReachedNotification extends Notification {
+
+    private long budgetId;
+
+    private String budgetName;
+
+    public LimitReachedNotification() {
+        this.setNotificationType(NotificationType.info);
+    }
+
+    public long getBudgetId() {
+        return budgetId;
+    }
+
+    public void setBudgetId(long budgetId) {
+        this.budgetId = budgetId;
+    }
+
+    public String getBudgetName() {
+        return budgetName;
+    }
+
+    public void setBudgetName(String budgetName) {
+        this.budgetName = budgetName;
+    }
+}
```

Comment:
If you use the annotations `@Data` or `@Getter` and `@Setter` you don't need to write getters and setters by yourself.